src/Components/Notification.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+export const Notification: React.FC = () => {
+  const [isButtonDelete, setIsButtonDelete] = useState(true);
```

Comment:
Consider renaming 'isButtonDelete' to a more descriptive name, such as 'isNotificationVisible' 🤔